projects/1.18/assets/chemlib/chemlib/lang/zh_cn.json

```
@@ -1 +1,407 @@
+  "item.chemlib.ingot_cesium": "铯锭",
+  "item.chemlib.ingot_barium": "钡锭",
+  "item.chemlib.ingot_lanthanum": "镧锭",
+  "item.chemlib.ingot_cerium": "铈锭",
+  "item.chemlib.ingot_praseodymium": "镨锭",
+  "item.chemlib.ingot_neodymium": "钕锭",
+  "item.chemlib.ingot_promethium": "钷锭",
+  "item.chemlib.ingot_samarium": "钐锭",
+  "item.chemlib.ingot_europium": "铕锭",
+  "item.chemlib.ingot_gadolinium": "钆锭",
+  "item.chemlib.ingot_terbium": "铽锭",
+  "item.chemlib.ingot_dysprosium": "镝锭",
+  "item.chemlib.ingot_holmium": "钬锭",
+  "item.chemlib.ingot_erbium": "铒锭",
+  "item.chemlib.ingot_thulium": "铥锭",
+  "item.chemlib.ingot_ytterbium": "镱锭",
+  "item.chemlib.ingot_lutetium": "镥锭",
+  "item.chemlib.ingot_hafnium": "铪锭",
+  "item.chemlib.ingot_tantalum": "钽锭",
+  "item.chemlib.ingot_tungsten": "钨锭",
+  "item.chemlib.ingot_rhenium": "铼锭",
+  "item.chemlib.ingot_osmium": "锇锭",
+  "item.chemlib.ingot_iridium": "铱锭",
+  "item.chemlib.ingot_platinum": "铂锭",
+  "item.chemlib.ingot_thallium": "铊锭",
+  "item.chemlib.ingot_lead": "铅锭",
+  "item.chemlib.ingot_bismuth": "铋锭",
+  "item.chemlib.ingot_polonium": "钋锭",
+  "item.chemlib.ingot_astatine": "砹锭",
+  "item.chemlib.ingot_francium": "钫锭",
+  "item.chemlib.ingot_radium": "镭锭",
+  "item.chemlib.ingot_actinium": "锕锭",
+  "item.chemlib.ingot_thorium": "钍锭",
+  "item.chemlib.ingot_protactinium": "镤锭",
+  "item.chemlib.ingot_uranium": "铀锭",
+  "item.chemlib.ingot_neptunium": "镎锭",
+  "item.chemlib.ingot_plutonium": "钚锭",
+  "item.chemlib.ingot_americium": "镅锭",
+  "item.chemlib.ingot_curium": "锔锭",
+  "item.chemlib.ingot_berkelium": "锫锭",
+  "item.chemlib.ingot_californium": "锎锭",
+  "item.chemlib.ingot_einsteinium": "锿锭",
+  "item.chemlib.ingot_fermium": "镄锭",
+  "item.chemlib.ingot_mendelevium": "钔锭",
+  "item.chemlib.ingot_nobelium": "锘锭",
+  "item.chemlib.ingot_lawrencium": "铹锭",
+  "item.chemlib.ingot_rutherfordium": "𬬻锭",
+  "item.chemlib.ingot_dubnium": "𬭊锭",
+  "item.chemlib.ingot_seaborgium": "𬭳锭",
+  "item.chemlib.ingot_bohrium": "𬭛锭",
+  "item.chemlib.ingot_hassium": "𬭶锭",
+  "item.chemlib.ingot_meitnerium": "鿏锭",
+  "item.chemlib.ingot_darmstadtium": "𫟼锭",
+  "item.chemlib.ingot_roentgenium": "𬬭锭",
+  "item.chemlib.ingot_copernicium": "鿔锭",
+  "item.chemlib.ingot_nihonium": "鉨锭",
+  "item.chemlib.ingot_flerovium": "𫓧锭",
+  "item.chemlib.ingot_moscovium": "镆锭",
+  "item.chemlib.ingot_livermorium": "𫟷锭",
+  "item.chemlib.ingot_tennessine": "Tennessine锭",
+  "item.chemlib.ingot_oganesson": "Oganesson锭",
+  "item.chemlib.compound_carbon_dioxide": "二氧化碳",
+  "item.chemlib.compound_cellulose": "纤维素",
+  "item.chemlib.compound_aluminum_oxide": "氧化铝",
+  "item.chemlib.compound_magnesium_oxide": "氧化镁",
+  "item.chemlib.compound_potassium_nitrate": "硝酸钾",
+  "item.chemlib.compound_silicon_dioxide": "二氧化硅",
+  "item.chemlib.compound_potassium_chloride": "氯化钾",
+  "item.chemlib.compound_sodium_chloride": "氯化钠",
+  "item.chemlib.compound_water": "水",
+  "item.chemlib.compound_kaolinite": "高岭土",
```

```suggestion
  "item.chemlib.compound_kaolinite": "高岭石",
```
https://www.termonline.cn/word/569980/1#s1